src/app/models/field/numberField.ts

```diff
@@ -1,21 +1,19 @@
 import { Schema } from 'mongoose'
 
-import { INumberFieldSchema, NumberSelectedValidation } from '../../../types'
+import {
+  INumberFieldSchema,
+  NumberSelectedValidation,
+  NumberValidationOptions,
+} from '../../../types'
+import { WithCustomMinMax } from '../../../types/field/utils/virtuals'
 
 import { MyInfoSchema } from './baseField'
 
 const createNumberFieldSchema = () => {
-  return new Schema<INumberFieldSchema>({
-    myInfo: MyInfoSchema,
-    ValidationOptions: {
-      customMax: {
-        type: Number,
-        default: null,
-      },
-      customMin: {
-        type: Number,
-        default: null,
-      },
+  const ValidationOptionsSchema = new Schema<
+    WithCustomMinMax<NumberValidationOptions>
```

Comment:
thanks! I added a todo for the type itself at `src/types/field/utils/virtuals.ts`, so I think it is ok to leave out here because once we delete the type, we would remove references to the type as well